plugins/webhooks/index.js

```diff
@@ -232,6 +237,21 @@ async function createPREvents(options, request) {
     return Promise.all(events);
 }
 
+/**
+ * Resolve ChainPR flag
+ * @method resolveChainPR
+ * @param  {Boolean}  chainPR              Plugin Chain PR flag
+ * @param  {Pipeline} pipeline             Pipeline
+ * @param  {Object}   pipeline.annotations Pipeline-level annotations
+ * @return {Boolean}
+ */
+function resolveChainPR(chainPR, pipeline) {
+    const defaultChainPR = typeof chainPR === 'undefined' ? false : chainPR;
```

Comment:
does this work too?
```suggestion
    const defaultChainPR = chainPR || false;
```

Reply:
Yes, but we should not use `||` short circuit evaluation about **Boolean** type.
Please refer to the following comments too.